cpp/src_prims/selection/knn.cuh

```diff
@@ -421,8 +421,8 @@ __global__ void regress_avg_kernel(LabelType *out, const int64_t *knn_indices,
  * @param y vector of label arrays. for multulabel classification,
  *          each output in the vector is a different array of labels
  *          corresponding to the i'th output.
- * @param n_rows number of rows in knn_indices
- * @param n_labels number of vertices in index (eg. size of each y array)
+ * @param n_query_rows number of rows in knn_indices
```

Comment:
should these comment lines match the order of arguments or does doxygen not care?